AntWars/Board/Board.cs

```diff
@@ -25,6 +25,8 @@ class Board
         public int Diagonal { get; private set; }
         internal Config conf;
         private CoordsInView[] coordsInViews = new CoordsInView[20];
+
+        private Coordinates[] baseCoords = new Coordinates[2];
```

Comment:
Klassen variablen sollten nur da sein wenn die variable entweden in mehreren methoden genutz wird oder öfters.
du könntest da wo generate base aufgerufen wird die beiden coords mitgeben, dann brauchst du das nicht als klassenvariable